wiki/FAQ/en.md

```diff
@@ -393,6 +393,10 @@ For any questions regarding pp, please refer to the [wiki page for pp first](/wi
 
 ### General
 
+#### I think I've found a bug! Where can I report it?
+
+If you have a general technical issue, or if you're unsure whether or not the issue you've encountered is a bug or not, you can post your problems on the [Help forums](https://osu.ppy.sh/community/forums/5). If you think you've found a specific, critical issue with osu!, you can create an issue on the [osu-stable GitHub page](https://github.com/ppy/osu-stable-issues).
```

Comment:
```suggestion
If you have a general technical issue, or if you're unsure whether or not the issue you've encountered is a bug or not, you can post your problems on the [Help forums](https://osu.ppy.sh/community/forums/5). If you think you've found a specific, critical issue with osu!, you can create an issue on the [osu-stable-issues GitHub repository](https://github.com/ppy/osu-stable-issues).
```
Suggestion, this wording is also present [here](https://osu.ppy.sh/community/forums/topics/338103).

Reply:
Yeah, but I doubt many people would default going to the forums street gimme finding a big.